R/qza_to_tse.R

```diff
@@ -0,0 +1,108 @@
+#' Generates a TreeSummarizedExperiment (TSE) object from .qza artifacts
+#'\code{\link[TreeSummarizedExperiment:SummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'
+#' Construct a TreeSummarizedExperiment object from multiple 
+#' qiime2 artifacts (.qza). 
+#' '\code{\link[TreeSummarizedExperiment:SummarizedExperiment-class]{TreeSummarizedExperiment}}
+#' Embedded metadata for provenance is not maintained in this function and 
+#' instead \code{read_qza()} should be used.
+#' 
+#' @param features file path for artifact containing a feature (OTU/SV) table
+#' @param tree file path for artifact containing a tree
+#' @param taxonomy file path for artifact containg taxonomy
+#' @param metadata file path for a qiime2-compliant TSV metadata file
+#' @param tmp a temporary directory that the object will be decompressed to.
+#' @return a TreeSummarizedExperiment object
+#' \code{\link[TreeSummarizedExperiment:SummarizedExperiment-class]{TreeSummarizedExperiment}}
```

Comment:
here you mention TreeSE twice, remove first one and replace it with the latter (linked) one

Reply:
removed the first instance